controllers/verification.go

```diff
@@ -108,13 +108,12 @@ func (c *ApiController) SendVerificationCode() {
 			c.ResponseError("Invalid phone number")
 			return
 		}
-		org := object.GetOrganization(orgId)
-		if org == nil {
+		if organization == nil {
 			c.ResponseError("Missing parameter.")
```

Comment:
```go
c.ResponseError("Missing parameter.")
```

Tell what param is missing like:

![image](https://user-images.githubusercontent.com/3787410/178113154-26257710-0be5-4ab1-826b-302ee865efc2.png)